src/Core/System/Snippet/Files/SnippetFileLoader.php

```diff
@@ -34,6 +39,18 @@ public function loadSnippetFilesIntoCollection(SnippetFileCollection $snippetFil
 
     private function loadPluginSnippets(SnippetFileCollection $snippetFileCollection): void
     {
+        try {
+            /** @var array<string, string> $authors */
+            $authors = $this->connection->fetchAllKeyValue('
+                SELECT `base_class` AS `baseClass`, `author`
+                FROM `plugin`
+            ');
+
+            $this->authors = $authors;
```

Comment:
You need to reset the data with the ResetInterface from symfony. otherwise we keep stuff in memory for ever.

Reply:
Oh yeah, sorry I missed that. Now it's added.